code/__DEFINES/preferences.dm

```diff
@@ -131,3 +131,9 @@
 	"End of Round Deathmatch" = BE_DEATHMATCH,\
 	"Prefer Squad over Role" = BE_SQUAD_STRICT\
 	)
+
+///Whether the client wants to be alerted to injured and badly hurt xenos.
+#define ALERTS_XENO_HEALTH	(1<<0)
+
+#define TOGGLES_ALERTS_DEFAULT (ALERTS_XENO_HEALTH)
+
```

Comment:
Default to off

Reply:
I don't know about this; if people find them annoying, it is very easy to turn off the alerts